core/modules/map.rs

```diff
@@ -48,14 +51,11 @@ use super::AssertedModuleType;
 type PrepareLoadFuture =
   dyn Future<Output = (ModuleLoadId, Result<RecursiveModuleLoad, Error>)>;
 
-use super::default_import_meta_resolve_cb;
```

Comment:
What happened to this guy?

Reply:
I removed the `Default` for ModuleMap as part of this change which means it is no longer necessary.

Reply:
So what if I use `FsModuleLoader`? `import.meta.resolve` will not work anymore?

Reply:
Sorry, I should be clearer -- the default value was already handled in JsRuntime. The ModuleMap default was never really necessary:

```
    let import_meta_resolve_cb = options
      .import_meta_resolve_callback
      .unwrap_or_else(|| Box::new(default_import_meta_resolve_cb));
```

Reply:
Ah, got it. My bad then